src/Rokt-Kit.js

```diff
@@ -398,14 +398,15 @@ function parseSettingsString(settingsString) {
     try {
         return JSON.parse(settingsString.replace(/&quot;/g, '"'));
     } catch (error) {
-        throw new Error('Settings string contains invalid JSON');
+        console.error('Settings string contains invalid JSON');
     }
+    return [];
 }
 
 function extractRoktExtensions(settingsString) {
-    var settings = settingsString ? parseSettingsString(settingsString) : [];
-
+    var settings = parseSettingsString(settingsString);
```

Comment:
I think this should still have the conditional.  otherwise everyone immediately will start to see a console.error about there not being invalid json right?, right?